src/components/SearchBar.tsx

```diff
@@ -364,7 +364,7 @@ export function SearchBar({
     || (!isVertical && filteredRecentSearches?.length) || entityPreviews);
   const screenReaderText = getScreenReaderText(
     autocompleteResponse?.results.length,
-    filteredRecentSearches?.length,
+    isVertical ? 0 : filteredRecentSearches?.length,
```

Comment:
because we don't display recent searches in the dropdown on vertical pages, so it's confusing to have the screenreader count there when they're not visible/accessible